e2e/Dockerfile.node

```diff
@@ -0,0 +1,28 @@
+FROM golang:1.9.2-alpine AS builder
```

Comment:
Why do we need separate `Dockerfile` for e2e?

Reply:
Because we don't have lightweight docker image for building both node/client and ability to start them up.

Reply:
I agree, I see this `Dockerfile.node` as duplication of our official image.

Reply:
Which one? I don't see any official images of building client + node on alpine

Reply:
`mysteriumnetwork/mysterium-node:alpine`
`mysteriumnetwork/mysterium-client:alpine`

Reply:
Ha ha, as they are commited one hour ago :) I will use them - thanks

Reply:
This `Dockerfile.node` is unused now

Reply:
Removed